### DISCUSSION TITLE
is deployer 7 stable ?

### DISCUSSION THREAD

**Discussion by mybouhssina** [deployphp/deployer#2728] [Help needed]
Hi,
Your deployer 6 documentation says **This is documentation for Deployer 6.x, which is no longer actively maintained.** but the download page says **Get latest version: Download deployer.phar (6.8.0)** and the 'other versions' contains the deployer 7 beta releases, also the 7.x documentation is full of TODOs, so I'm kind of confused, is there a deployer 7 stable release ? where is its documentation?

**Comment by antonmedv** [comment]
Documentation is here: https://deployer.org/docs/7.x/basics

I'd say "beta" is stable. I think 10% of users already using beta. Will release sooooooon!!)

**Reply by mybouhssina** [reply]
Thank you for the blazing fast answer !
But the "main concepts" menu items (which are referenced from basics) are missing (contain a TODO), am I missing something ?

**Reply by antonmedv** [reply]
Yes,) docs now are missing. In process of writing it. Consider supporting me on GitHub to motivate me more!

**Reply by mybouhssina** [reply]
oh ok, thank you very much ! :)
